src/reducers/chartReducer.js

```diff
@@ -88,17 +89,13 @@ export const chartCursorAction = (cursorBegin, cursorEnd) => ({
     cursorEnd,
 });
 
-export const chartWindowAction = (
+const chartWindowAction2 = (
```

Comment:
[minor] Not a huge fan of just appending 2 to the name. Perhaps the initial function should be changed to `prepareChartWindowAction` and this one be called `chartWindowAction` or something like that?

Reply:
I didn't want to change the name of the function everywhere where it was called :\